src/ec/packages/ec-component-button/button.story.js

```diff
@@ -45,86 +83,114 @@ storiesOf('Components/Button', module)
   .addDecorator(withNotes)
   .add(
     'primary',
-    () =>
-      button(
-        merge(prepareButton(dataPrimary), {
-          icon: {
-            name: select('Icon (sample)', iconsList, null),
-          },
-          label: text('label', dataPrimary.label),
-          icon_position: select('Icon position', iconPositionSettings, 'after'),
-          disabled: boolean('Disabled', false),
-        })
-      ),
+    () => {
+      return button(prepareButton(primaryData));
+    },
     {
-      notes: { markdown: notes, json: prepareButton(dataPrimary) },
+      notes: { markdown: notes, json: { button: primaryData } },
```

Comment:
i don't see the "demo" input data in the notes being updated after touching the knobs.

Reply:
but it's still not good, @Joosthe 
![image](https://user-images.githubusercontent.com/14326885/75683693-e92bcb80-5c97-11ea-963f-b0a7f285e898.png)

Reply:
@Joosthe, sorry, but nothing has changed regarding this from what i see. Is there anything unclear?
Not sure it was already the case before, also, but if you look at the button without the icon you will see that the markup of the icon is in place and the space is taken, just that there is no icon with that "name".